packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -63,21 +63,34 @@ def __init__(
         :param contract_interface: the contract interface.
         """
         super().__init__(contract_id, contract_config, contract_interface)
-        self.token_ids = {}  # type: Dict[int, int]
+        self._token_ids = {}  # type: Dict[int, int]
+        self._game_currency_id = 0  # type: int
 
-    def create_token_ids(self, token_type: int, nb_tokens: int) -> Dict[int, int]:
+    @property
+    def token_ids(self) -> Dict[int, int]:
+        """The generated token ids."""
+        return self._token_ids
+
+    @property
```

Comment:
@Totoual I think this property and associated attribute should be removed. They should be kept on the business logic side of the skill, not here.

Reply:
From the perspective of the erc1155 they are all tokens, so the `token_ids` dict is enough